stretch-auth/src/secrets.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Nice job on the secret file!

Reply:
This definitely isn't hidden/secret

Reply:
We'll have to figure out how to truly hide this.